packages/pattern-picker/src/item.tsx

```diff
@@ -19,11 +20,11 @@ export function Item( { style, onClick, pattern, className }: Props ) {
 				className={ cx( 'pattern-picker__item', className ) }
 			>
 				<div className="pattern-picker__item-iframe-wrapper">
-					<iframe
-						frameBorder="0"
-						scrolling="no"
-						title={ pattern.name }
-						src={ `https://public-api.wordpress.com/wpcom/v2/block-previews/pattern?stylesheet=pub%2Flynx&pattern_id=${ pattern.name }` }
+					<MShotsImage
+						url={ `https://public-api.wordpress.com/wpcom/v2/block-previews/pattern?stylesheet=pub%2Flynx&pattern_id=${ pattern.name }` }
+						options={ { w: 512, vpw: 512, vph: 1120 } }
+						alt={ pattern.title }
```

Comment:
Just a generic note; for accessibility, we should ensure these descriptions are descriptive enough without actually seeing the pattern.

Reply:
Not really, it's hard-coded right now. The patterns API only returns all patterns (hundreds with their HTML inlined) so I decided to hardcode the 13 we use instead of loading all this data.

Reply:
That's fine for now but let's solve it once other priority stuff is sorted.